docs/cqd_reference/src/asciidoc/_chapters/query_execution.adoc

```diff
@@ -237,3 +237,54 @@ If you enable small scanner on large size scan incorrectly, then you are likely
 | *Deprecated In Release*     | Not applicable.
 |===
 
+<<<
+[[scratch-dirs]]
+== SCRATCH_DIRS
+
+[cols="25%h,75%"]
+|===
+| *Description*               |	Allows a CQD specification to override the envvar settings for scatch and temporary file locations, which gives user more flexibility to change scratch directory locations.
+| *Values*                    |
+Scratch disk locations for overflow (linux path names).
+
+Multiple locations can be specified, separated by a colon. For example, /tmp/scratch1:/tmp/scratch2.
+
+| *Usage*                     a|
+
+*Two ways to change scratch directory location permanently*:
+
+* Specify this location at installation time.
+
++
+Currently the scratch directory locations are set at installation time. The layer of scratch and temporary file system that manages the scratch file creation and reads the envvar in $TRAF_HOME/etc/ms.env to determine where to create the scratch files.
```

Comment:
Suggested wordsmith: "Currently the scratch directory locations are set at installation time. The code that manages scratch and temporary files reads environment variables in $TRAF_HOME/etc/ms.env to determine where to create scratch files."